### PR TITLE
Front end should be stable now

### PR DESCRIPTION
**Pull request by oldchevy**
-fix stream formatting
-adds transitions
-fix buttons
-include graph init in componentDidRender

### CODE REVIEW

File: Public/app/Components/NavBar.jsx [hunk 1]
```diff
@@ -11,9 +11,7 @@ var NavBar = ({logout, savePrefs, synced}) => (
           Try interacting with the ticker graphs on the right! In your account, you can customize your 
           dashboard view any way you like, and save those preferences so that you see them each time you log in. 
         </span>
-        <form action="/login" method="get">
-          <button className="btn btn-default"></button>
```
Comment (p-herbert) [inline Public/app/Components/NavBar.jsx:15]: Should this button have text?

File: Public/app/Components/NavBar.jsx [hunk 2]
```diff
@@ -11,9 +11,7 @@ var NavBar = ({logout, savePrefs, synced}) => (
...
-        </form>
+        <button className="btn btn-default" onClick={() => window.location ='/login'}>Logout</button>
```
Comment (oldchevy) [inline Public/app/Components/NavBar.jsx:14]: Gave it back its text right here

Comment (oldchevy) [inline Public/app/Components/NavBar.jsx:14]: And took away the form wrapping. Clarabelle and I were just conjuring with that form element

**Comment by p-herbert** [comment]
Ok looks good

**Pull request merged**